oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/IndexOptions.java

```diff
@@ -183,10 +197,16 @@ public int consistencyCheckLevel(){
         return consistencyCheck.value(options);
     }
 
+    public long aysncDelay() {
+        return  asyncDelay.value(options);
+    }
+
     public boolean isReindex() {
         return options.has(reindex);
     }
 
+    public boolean isAsyncIndex() {return options.has(asyncIndex);}
```

Comment:
Could you re-format (multiple lines, same as isReindex)?

Reply:
Done